Services/CustomDeviceModels.cs

```diff
@@ -42,16 +42,20 @@ public interface ICustomDeviceModels
     public class CustomDeviceModels : ICustomDeviceModels
     {
         private const string STORAGE_COLLECTION = "deviceModels";
+        private const string SERVICE_ERROR = "ServiceError";
```

Comment:
Do we have a common Constants file? If so, please move this constant there instead of repeating in every file

Reply:
there is no constants file for services. We do have one for tests but I don't think we might wanna leverage that.